scripts/pynag

```diff
@@ -690,6 +700,45 @@ def pretty_print(objects, attributes=None):
         pre = "-"*10
         post = "-"*(80-10-len(message))
         print pre + message + post
+
+def get_nagios_attributes(obj):
+    obj_attributes = {}
+    for attribute, value in obj.items():
+        # Don't return pynag internal attributes
+        # except the object type
+        if attribute == "id":
+            continue
+        if attribute == "meta":
+            obj_attributes["object_type"] = value['object_type']
+            continue
+        # or attributes with no value
+        if not value:
```

Comment:
value could explicitly be set to empty string, meaning 'dont inherit from template', and it can be subtly different from no value at all when the nagios is done parsing the config.

Better replace this with 'if value is None:'